crates/api_models/src/payments.rs

```diff
@@ -1814,7 +1814,9 @@ pub enum AdditionalPaymentData {
     Wallet {
         apple_pay: Option<ApplepayPaymentMethod>,
     },
-    PayLater {},
+    PayLater {
+        klarna_sdk: Option<KlarnaSdkPaymentMethod>,
+    },
```

Comment:
Can you ensure that this change is compatible for reverts? Create the payment with new version and try to retrieve it from the older version?

Reply:
Do we have an update on this @swangi-kumari ?

Reply:
Yes @Narayanbhat166 , I tried creating the payment with new version and was able to retrieve it from the older version. It is compatible.
cc- @SamraatBansal

Reply:
Tested the same, getting expected response.

psync response:
```
"payment_method_data": {
        "pay_later": {
            "klarna_sdk": null
        },
        "billing": null
    }
```